foundation/plugins/org.jboss.tools.foundation.help.ui/src/org/jboss/tools/foundation/help/ui/internal/search/SearchiskoResultBuilder.java

```diff
@@ -77,11 +80,20 @@ public String toAbsoluteHref(String href, boolean frames) {
 		}
 		
```

Comment:
Please document behavior, that if the type is a list, it uses first item in the list.